README.md

```diff
@@ -1,56 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
+# Stock market
 
+> "Stock Market Metrics" is a website wher you can find information abput a cpmpany financial statement in the previous years.
 ## Built With
 
 - Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] your project is missing Live version link as requested in the project guidelines.

Reply:
hey @arslanbisharat thank you for spotting this. i have added the live link